proto/dydxprotocol/vault/genesis.proto

```diff
@@ -18,6 +18,8 @@ message GenesisState {
   repeated Vault vaults = 3 [ (gogoproto.nullable) = false ];
   // The default quoting parameters for all vaults.
   QuotingParams default_quoting_params = 4 [ (gogoproto.nullable) = false ];
+  // All locked shares.
```

Comment:
nit: Add a note to `total_shares` about whether it includes or excludes locked shares.